### DISCUSSION TITLE
Could this work on the same server with deployer also installed?

### DISCUSSION THREAD

**Discussion by vivgui** [webpod/webpod#7] [Q&A]
I'm running a laravel application and deploy it with deployer. I also have an astro site I want to use it for the marketing site of the app. While I do have options like netlify, vercel, etc, I wanted to know if its possible to use deployer and webpod on the same server? That way, everything would be centralized.

**Comment by antonmedv** [comment]
It is possible. Provision with webpod and only use deploy recipe from deployer.

**Comment by vivgui** [comment]
So you mean, provision the server using webpod but deploy the static site and laravel site using deployer?

**Reply by antonmedv** [reply]
yes
